src/main/java/com/agilethought/atsceapi/model/UserModel.java

```diff
@@ -0,0 +1,46 @@
+package com.agilethought.atsceapi.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "USERS")
+public class UserModel {
```

Comment:
I prefer call this class User, in that way we can mapping with one collection into DB as `users`. 
Althoug, we can infer that this is a `model` because of java package.